src/buffer/out/AttrRowIterator.cpp

```diff
@@ -123,14 +130,23 @@ void AttrRowIterator::_decrement(size_t count)
 {
     while (count > 0)
     {
+        // If there's still space within this color attribute to move left, do so.
         if (count <= _currentAttributeIndex)
         {
             _currentAttributeIndex -= count;
             return;
         }
+        // If there's not space, move the whole color attribute left by one
+        // and just take one away from the remaining movement.

```

Comment:
this comment doesn't match the math below -- it's no longer just subtracting one